docs/content/installation/install_hostdefiner.md

```diff
@@ -0,0 +1,13 @@
+# Installing the host definer
+
+Install the HostDefiner custom resource to enable dynamic host definitions of the CSI driver nodes.
+
+For more information on using dynamic host connectivity definition, see [Using dynamic host definition](../using/using_hostdefinition.md).
+
+For information on configuring the HostDefiner custom resource, see [Configuring the host definer](../configuration/configuring_hostdefiner).
+
+The host definer can be downloaded and installed from the OpenShift web console and OperatorHub.io, as part of the operator and driver installation process. For more information, see [Installing the driver with the OpenShift web console](install_driver_openshift_web.md) and [Installing the driver with OperatorHub.io](install_driver_operatorhub.md).
+
+The host definer can also be installed directly with GitHub (see [Installing the host definer with GitHub](install_hostdefiner_github.md)).
```

Comment:
I don't think we should maintain the types of installations here (it just duplicates the information for both the reader and the writer), and I don't see why we should have this arbitrary separation of the GitHub method.
```suggestion
The host definer can be downloaded and installed as part of the operator and driver installation process. For more information, see [Installing the operator and driver](install_operator_driver.md)
```

Reply:
they're separate installations, and as you've pointed out, the host definer is not part of the driver and is optional and separate.
from a documentation point of view they should be kept separate. I've included information within the "main" sections as much as the procedures allowed / it made sense but other than that this should be documented separately.

Reply:
I still don't see how the GitHub one is any different in this aspect.
Both the driver and the host definer can be installed (after installing the operator) very similarly, in all installation methods.
I don't understand what could be missed by unifiying the GitHub installations as well

Reply:
with the way ibm documentation is going, we want to keep sections separate, when possible, as it's per use-case basis. while it _could_ in theory be put together, as you suggest, it is easier for searching, finding, and later transitioning into the new documentation guidelines that are being developed. 
the installation types must be documented separately, especially as it doesn't include satellite currently.

Please close this thread. the information is being kept separate in its own section. thanks